### PR TITLE
Added basic support for aerisdies.com

### PR DESCRIPTION
**Pull request by cyian-1756**
# Category

This change is exactly one of the following (please change `[ ]` to `[x]`) to indicate which:
* [ ] a bug fix -- which issue? #...
* [X] a new Ripper
* [ ] a refactoring
* [ ] a style change/fix

# Description

A ripper for aerisdies.com.

Test link: http://www.aerisdies.com/html/lb/douj_11369_1.html

# Testing

Required verification:
* [X] I've verified that there are no regressions in `mvn test` (there are no new failures or errors).
* [X] I've verified that this change works as intended.
  * [X] Downloads all relevant content.
  * [X] Downloads content from multiple pages (as necessary or appropriate).
  * [X] Saves content at reasonable file names (e.g. page titles or content IDs) to help easily browse downloaded content.
* [X] I've verified that this change did not break existing functionality (especially in the Ripper I modified).

Optional but recommended:
* [ ] I've added a unit test to cover my change.

### CODE REVIEW

File: src/main/java/com/rarchives/ripme/ripper/rippers/AerisdiesRipper.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        return "aerisdies";
+    }
+    @Override
+    public String getDomain() {
+        return "aerisdies.com";
+    }
+
+    @Override
+    public String getGID(URL url) throws MalformedURLException {
+        Pattern p = Pattern.compile("^https?://www.aerisdies.com/html/lb/([a-z]*_[0-9]*_\\d)\\.html");
```
Comment (rautamiekka) [inline src/main/java/com/rarchives/ripme/ripper/rippers/AerisdiesRipper.java]: IMO `.html` should be `.html?` to avoid a possible problem.

Comment (cyian-1756) [inline src/main/java/com/rarchives/ripme/ripper/rippers/AerisdiesRipper.java]: I'm not that good with regex what would add `?` do? Wouldn't it just match `htm` and `html`?

Comment (rautamiekka) [inline src/main/java/com/rarchives/ripme/ripper/rippers/AerisdiesRipper.java]: Yes, exactly that.

Comment (cyian-1756) [inline src/main/java/com/rarchives/ripme/ripper/rippers/AerisdiesRipper.java]: But why? Like what use case would cause the `l` to be missing from `.html`?

Comment (rautamiekka) [inline src/main/java/com/rarchives/ripme/ripper/rippers/AerisdiesRipper.java]: I can't think of any use cases, but not everyone writes the full extension.

Comment (cyian-1756) [inline src/main/java/com/rarchives/ripme/ripper/rippers/AerisdiesRipper.java]: Wouldn't most people copy and paste the link? Also wouldn't a link missing the l in html just 404? I just don't really see anything to gain by matching malformed/incomplete links

Comment (rautamiekka) [inline src/main/java/com/rarchives/ripme/ripper/rippers/AerisdiesRipper.java]: I mean the hosts.

By default you'd get 404, yes, just tested.

**Pull request merged**